tslint.json

```diff
@@ -1,221 +1,191 @@
+        "no-typeof-undefined": true,
+        "no-unnecessary-bind": true,
+        "no-unnecessary-class": true,
+        "no-unnecessary-field-initialization": true,
+        "no-unnecessary-local-variable": true,
+        "no-unnecessary-override": true,
+        "no-unnecessary-semicolons": true,
+        "no-unnecessary-type-assertion": false,
+        "no-unsupported-browser-code": true,
+        "no-useless-files": true,
+        "no-with-statement": true,
+        "non-literal-fs-path": true,
+        "non-literal-require": true,
+        "number-literal-format": true,
+        "possible-timing-attack": true,
+        "prefer-array-literal": true,
+        "prefer-conditional-expression": false,
+        "prefer-object-spread": true,
+        "prefer-switch": false,
+        "prefer-type-cast": true,
+        "promise-must-complete": true,
+        "react-a11y-anchors": true,
+        "react-a11y-aria-unsupported-elements": true,
+        "react-a11y-event-has-role": true,
+        "react-a11y-image-button-has-alt": true,
+        "react-a11y-img-has-alt": true,
+        "react-a11y-input-elements": true,
+        "react-a11y-lang": true,
+        "react-a11y-meta": true,
+        "react-a11y-no-onchange": true,
+        "react-a11y-props": true,
+        "react-a11y-proptypes": true,
+        "react-a11y-required": true,
+        "react-a11y-role-has-required-aria-props": true,
+        "react-a11y-role-supports-aria-props": true,
+        "react-a11y-role": true,
+        "react-a11y-tabindex-no-positive": true,
+        "react-a11y-titles": true,
+        "react-anchor-blank-noopener": true,
+        "react-iframe-missing-sandbox": true,
+        "react-no-dangerous-html": true,
+        "react-this-binding-issue": true,
+        "react-unused-props-and-state": true,
+        "space-within-parens": true,
+        "switch-final-break": true,
+        "type-literal-delimiter": false,
+        "underscore-consistent-invocation": true,
+        "use-default-type-parameter": false,
+        "use-named-parameter": true,
 
-    // tslint-microsoft-contrib rules disabled
-    "missing-jsdoc": false,
-    "no-duplicate-case": false,
-    "no-empty-interfaces": false,
-    "no-empty-line-after-opening-brace": false,
-    "no-multiline-string": false,
-    "no-relative-imports": false,
-    "no-stateless-class": false,
-    "no-unexternalized-strings": false,
-    "no-var-self": false,
-    "react-tsx-curly-spacing": false,
-    "valid-typeof": false
-  }
+        // tslint-microsoft-contrib rules disabled
+        "missing-jsdoc": false,
+        "no-duplicate-case": false,
+        "no-empty-interfaces": false,
+        "no-empty-line-after-opening-brace": false,
+        "no-multiline-string": false,
+        "no-relative-imports": false,
+        "no-stateless-class": false,
+        "no-unexternalized-strings": false,
+        "no-var-self": false,
+        "react-tsx-curly-spacing": false,
+        "valid-typeof": false
+    }
 }
```

Comment:
These rules can be completely removed from `tslint.json` because they're mentioned in `tslint-config-prettier`, right?

max-line-length
object-literal-key-quotes
quotemark
trailing-comma
whitespace
align
arrow-parens
arrow-return-shorthand (?)
space-before-function-name
space-before-function-paren

Reply:
Yes, it can be removed.